packages/theming/apple-theme/src/appleColors.macos.ts

```diff
@@ -332,40 +325,40 @@ export function fallbackApplePalette(): ThemeColorDefinition {
-    buttonTextChecked: fluentUIApple.neutralForeground3,
-    buttonTextCheckedHovered: fluentUIApple.neutralForeground3,
-    buttonTextPressed: ColorWithSystemEffectMacOS(fluentUIApple.neutralForeground3, 'pressed'),
-    buttonTextDisabled: ColorWithSystemEffectMacOS(fluentUIApple.neutralForeground3, 'disabled'),
-
-    buttonBorderDisabled: 'transparent',
-    buttonBorderFocused: 'transparent',
+    buttonBackground: macOSAliasColorTokens.neutralBackground3,
+    buttonBackgroundChecked: macOSAliasColorTokens.neutralBackground3,
+    buttonBackgroundHovered: macOSAliasColorTokens.neutralBackground3,
+    buttonBackgroundCheckedHovered: macOSAliasColorTokens.neutralBackground3,
+    buttonBackgroundPressed: ColorWithSystemEffectMacOS(macOSAliasColorTokens.neutralBackground3, 'pressed'),
+    buttonBackgroundDisabled: ColorWithSystemEffectMacOS(macOSAliasColorTokens.neutralBackground3, 'disabled'),
+    buttonBorder: macOSAliasColorTokens.transparentBackground,
+    buttonText: macOSAliasColorTokens.neutralForeground3,
+    buttonTextHovered: macOSAliasColorTokens.neutralForeground3,
+    buttonTextChecked: macOSAliasColorTokens.neutralForeground3,
+    buttonTextCheckedHovered: macOSAliasColorTokens.neutralForeground3,
+    buttonTextPressed: ColorWithSystemEffectMacOS(macOSAliasColorTokens.neutralForeground3, 'pressed'),
+    buttonTextDisabled: ColorWithSystemEffectMacOS(macOSAliasColorTokens.neutralForeground3, 'disabled'),
+
+    buttonBorderDisabled: macOSAliasColorTokens.transparentBackground,
+    buttonBorderFocused: macOSAliasColorTokens.transparentBackground,
 
     primaryButtonBackground: fluentUIApple.communicationBlue,
     primaryButtonBackgroundHovered: fluentUIApple.communicationBlue,
     primaryButtonBackgroundPressed: ColorWithSystemEffectMacOS(fluentUIApple.communicationBlue, 'pressed'),
     primaryButtonBackgroundDisabled: fluentUIApple.brandBackgroundDisabled,
     primaryButtonBorder: 'transparent',
     primaryButtonBorderFocused: 'transparent',
-    primaryButtonText: fluentUIApple.neutralInverted,
-    primaryButtonTextHovered: fluentUIApple.neutralInverted,
-    primaryButtonTextPressed: ColorWithSystemEffectMacOS(fluentUIApple.neutralInverted, 'pressed'),
+    primaryButtonText: macOSAliasColorTokens.neutralForegroundInverted,
+    primaryButtonTextHovered: macOSAliasColorTokens.neutralForegroundInverted,
+    primaryButtonTextPressed: ColorWithSystemEffectMacOS(macOSAliasColorTokens.neutralForegroundInverted, 'pressed'),
     primaryButtonTextDisabled: fluentUIApple.brandForegroundDisabled,
 
     accentButtonBackground: fluentUIApple.communicationBlue,
-    accentButtonText: fluentUIApple.neutralInverted,
+    accentButtonText: macOSAliasColorTokens.neutralForegroundInverted,
 
     menuBackground: 'transparent',
     menuDivider: applePlatform.separatorColor,
-    menuIcon: neutralForeground3, //GH:728 Icon doesn't support PlatformColor
+    menuIcon: macOSAliasColorTokens.neutralForeground3,
```

Comment:
@Saadnajmi is it appropriate to not use PlatformColor here and use alias tokens instead?

Reply:
Eventually the token pipeline should have support for platform color, such that this token _is_ a platform color. This specific token is used in svgs so it can't be platform color just yet though.

Reply:
I would keep the comment with the bug number

Reply:
Makes sense, fixed in d2132188efc7e7c453b4eaccff05d9b3f96dd578